config/target.exs

```diff
@@ -28,10 +29,13 @@ keys =
 
 if keys == [],
   do:
-    Mix.raise("""
-    No SSH public keys found in ~/.ssh. An ssh authorized key is needed to
-    log into the Nerves device and update firmware on it using ssh.
-    See your project's config.exs for this error message.
+    IO.write(:stderr, """
+    Warning: No SSH public key found. You will not be able to remotely
+    connect to the device to access the iex shell or updating firmware.
+
+    If you have an SSH public key, you can set NERVES_SSH_PUB_KEY to
+    the path of the key. Otherwise, you can still flash the firmware
+    manually and access the iex shell using a keyboard and monitor.
```

Comment:
"access the iex shell using a keyboard and monitor."

How would this be possible?  Do you mean plugging into the HDMI port? That may be true, but there's no USB host port for a keyboard.  The only two ports are a gadget port and power only port...

Reply:
That's what I get for seeing the second micro usb port and not realizing it's power only.  Updating.  Thanks!